src/App.scss

```diff
@@ -1,7 +1,4 @@
-@import "colors";
-
-.App {
-  width: 100%;
-  height: 100vh;
-  background: $common-background;
+* {
+  padding: 0;
+  margin: 0;
 }
```

Comment:
you sure you want to use a * selector? It's kinda dangerous but i won't mind it if @nsukhotskaya doesn't mind

Reply:
I agree with you. Better to avoid this approach.